### DISCUSSION TITLE
Connect to a random country

### DISCUSSION THREAD

**Discussion by mat926** [qdm12/gluetun#1915] [Q&A]
Is it possible to choose any random country every time the VPN connects? If I leave the SERVER_COUNTRIES variable blank then I expect it should randomly choose any available country from the VPN provider.

**Comment by frepke** [comment]
Yes, that's right (myself, I don't use the variable `COUNTRIES` at all).

**Comment by miklosbagi** [comment]
Is that not the standard behavior?
I'm using testing this with PIA, setting SERVER_REGIONS="Bulgaria,France,Kazakhstan,UK London,FI Helsinki,France,Norway", it picks a different country every time I kill it (well, instruct via control port to break the connection. I don't establish COUNTRIES at all.  
Is that what you're after?
